bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -298,10 +287,13 @@ public CompositeData getStatistics() throws OpenDataException {
     public void closeAllResolvers() {
         if (!resolvers.isEmpty()) {
             for (ReusableResolver resolver : resolvers) {
-                resolver.getResolver().close();
+                if (resolver.getResolver().isLive()) {
```

Comment:
This should be closed regardless of `.isLive()` check .. .javadocs are misleading. see @cziegeler note on the sling lists: 

http://apache-sling.73963.n3.nabble.com/Using-the-resource-resolver-td4061854.html